filecopy/operations/services/approval/client.py

```diff
@@ -1,83 +1,33 @@
-from operations.services.approval.models import CopyStatus
-from sqlalchemy import Column
-from sqlalchemy import MetaData
-from sqlalchemy import Table
-from sqlalchemy import select
-from sqlalchemy import update
-from sqlalchemy.dialects.postgresql import UUID
-from sqlalchemy.future import Engine
+from requests import Session
+
+logger = logging.getLogger(__name__)
 
 
 class ApprovalServiceClient:
     """Get information about approval request or entities for copy request."""
 
-    def __init__(self, engine: Engine, metadata: MetaData) -> None:
-        self.engine = engine
-        self.metadata = metadata
-
-    def _load_table(self, name: str) -> Table:
-        return Table(
-            name,
-            self.metadata,
-            Column('id', UUID(as_uuid=True), unique=True, primary_key=True, default=uuid4),
-            keep_existing=True,
-            autoload_with=self.engine,
-        )
-
-    @cached_property
-    def approval_entity(self) -> Table:
-        return self._load_table('approval_entity')
-
-    @cached_property
-    def approval_request(self) -> Table:
-        return self._load_table('approval_request')
-
-    def get_approval_request(self, request_id: str) -> ApprovalRequest:
-        """Return approval request by id."""
-
-        statement = select(self.approval_request).filter_by(id=request_id)
-        cursor = self.engine.connect().execute(statement)
-
-        approval_request = ApprovalRequest.from_orm(cursor.fetchone())
+    def __init__(self, endpoint: str, request_id: str) -> None:
+        self.endpoint = f'{endpoint}/v1'
+        self.request_id = request_id
+        self.client = Session()
 
-        return approval_request
-
-    def get_approval_entities(self, request_id: str) -> ApprovalEntities:
-        """Return all approval entities related to request id."""
-
-        statement = select(self.approval_entity).filter_by(request_id=request_id)
-        cursor = self.engine.connect().execute(statement)
-
-        request_approval_entities = ApprovalEntities.from_cursor(cursor)
-
-        return request_approval_entities
-
-    def update_copy_status(self, approval_entity: ApprovalEntity, copy_status: CopyStatus) -> None:
+    def update_copy_status(self, entity_id: str) -> None:
         """Update copy status field for approval entity."""
-
-        statement = (
-            update(self.approval_entity)
-            .where(self.approval_entity.columns.id == approval_entity.id)
-            .values(copy_status=copy_status)
-        )
-
-        with self.engine.begin() as connection:
-            connection.execute(statement)
+        payload = {'entities': [entity_id], 'copied_status': 'copied'}
+        response = self.client.put(f'{self.endpoint}/request/{self.request_id}/copy-status', json=payload)
+        if response.status_code != 200 or len(response.json()['result']) != 1:
+            raise Exception(f'Unable to update copy status for {entity_id}')
```

Comment:
Based on our discussion I would suggest to have here:
```suggestion
        if response.status_code == 200 and len(response.json()['result']) == 0:
            raise Exception(f'Unable to update copy status for {entity_id}, entity is not found.')
        if response.status_code != 200:
            raise Exception(f'Unable to update copy status for {entity_id}')
```
Because if we have `status_code != 200` this mean that response might have unexpected structure and `result` field might be missing, so we should trust response only if we have 200 status code.